Description: adapt to batik 1.8 API
 New path for class SAXSVGDocumentFactory.
Author: Gilles Filippini <pini@debian.org>
Index: scilab-5.5.2/modules/graph/src/java/org/scilab/modules/graph/utils/ScilabGraphUtils.java
===================================================================
--- scilab-5.5.2.orig/modules/graph/src/java/org/scilab/modules/graph/utils/ScilabGraphUtils.java
+++ scilab-5.5.2/modules/graph/src/java/org/scilab/modules/graph/utils/ScilabGraphUtils.java
@@ -28,7 +28,7 @@ import org.apache.batik.bridge.DocumentL
 import org.apache.batik.bridge.GVTBuilder;
 import org.apache.batik.bridge.UserAgent;
 import org.apache.batik.bridge.UserAgentAdapter;
-import org.apache.batik.dom.svg.SAXSVGDocumentFactory;
+import org.apache.batik.anim.dom.SAXSVGDocumentFactory;
 import org.apache.batik.gvt.GraphicsNode;
 import org.apache.batik.util.XMLResourceDescriptor;
 import org.scilab.forge.jlatexmath.ParseException;
